create-repo.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import re
+import subprocess
+import sys
+import textwrap
+
+import requests
+
+sparql_endpoint = 'https://query.wikidata.org/sparql'
+iso_3166_1_code = sys.argv[1].lower()
+
+if os.path.isdir(iso_3166_1_code):
+    raise AssertionError(f'Country repository already exists: {iso_3166_1_code}')
+
+query = f'''\
+SELECT * WHERE {{
+  ?country wdt:P297 '{iso_3166_1_code.upper()}' ;
+           wdt:P37/wdt:P424 ?language ;
+           rdfs:label ?label .
+  FILTER(LANG(?label) = 'en')
+}}
+'''
+
+print(query)
+
+response = requests .post(sparql_endpoint, query,
+                          headers = {'Content-Type': 'application/sparql-query',
+                                     'Accept': 'application/sparql-results+json'})
+response.raise_for_status()
+
+results = response.json()
+bindings = results['results']['bindings']
+
+languages = {binding['language']['value'] for binding in bindings}
+languages = sorted(languages)
+if 'en' not in languages:
+    languages.append('en')
+
+country_wikidata_id = bindings[0]['country']['value'].split('/')[-1]
+label = bindings[0]['label']['value']
+
+# Slugify the label for inclusion in the repository name:
+# * lowercase the English (en) country label
+# * drop apostrophes
+# * replace runs of non-letters with hyphens
+# * strip any leading or trailing hyphens
+repo_name = 'proto-commons-' + re.sub('[^a-z]+', '-', label.lower().replace("'", '')).strip('-')
+
+print(languages)
+print(country_wikidata_id)
+print(label)
+print(repo_name)
+
+os.mkdir(iso_3166_1_code)
+subprocess.check_call(['git', 'init'], cwd=iso_3166_1_code)
+
+files_for_initial_commit = ['config.json', 'Gemfile', 'Gemfile.lock']
+
+with open(os.path.join(iso_3166_1_code, 'config.json'), 'w') as f:
+    json.dump({
+        'country_wikidata_id': country_wikidata_id,
+        'languages': languages,
+    }, f, indent=2)
+
+for dirname in ['boundaries/build', 'executive', 'legislative']:
+    dirname = os.path.join(iso_3166_1_code, dirname)
```

Comment:
To make this slightly easier to read, I'd use a new variable name here rather than rebinding `dirname`.